tsconfig.json

```diff
@@ -4,14 +4,16 @@
         "target": "es6",
         "outDir": "out",
         "lib": [
-            "es6"
+            "es6",
+            "DOM"
```

Comment:
👀

Reply:
I know. I'm not happy we have to add that but I hope to be able to remove it once I move this whole feature onto built-in fetch.